api/src/main/java/openconsensus/stats/Measure.java

```diff
@@ -70,8 +70,16 @@
    */
   public abstract String getUnit();
 
-  // Prevents this class from being subclassed anywhere else.
-  Measure() {}
+  /**
+   * Returns a new {@link Measurement} for this {@code Measure}.
+   *
+   * @param value the corresponding value for the {@code Measurement}.
+   * @return a new {@link Measurement} for this {@code Measure}.
+   */
+  public final Measurement createMeasurement(double value) {
```

Comment:
should it return `Point` from metrics namespace and get rid of `Measurement` completely?

Reply:
Point represents an aggregated data, measurements represents one value for this measure.